Automattic/wp-calypso
# Pricing page rework: Add lightbox link to existing pricing page.

#### Proposed Changes

This PR is just an enhancement to #66751, where there was [a discussion](https://github.com/Automattic/wp-calypso/pull/66751#discussion_r950527560) about adding a "**More about <product_name>**" link into the pricing cards.

Looking into the code, I found a pretty simple and straightforward way to add the "**More about <product_name>**" link, like we were originally planed and discussed, so I went ahead and created this PR with the changes.

**[DO NOT MERGE] yet** - We still need to check with the Marketing team however, to determine if adding the lightbox link would be a helpful enhancement for current users, or would it possibly disrupt the product selection into checkout flow.

![Markup 2022-08-22 at 14 08 36](https://user-images.githubusercontent.com/11078128/185995516-57adc658-076a-4d35-ad6b-2a29e889e958.png)

![Markup 2022-08-22 at 14 09 45](https://user-images.githubusercontent.com/11078128/185995561-11272afd-cd20-4d9f-a7c5-710c04bb9701.png)

#### Testing Instructions

- Checkout the PR locally and start `yarn start-jetpack-cloud`.
- Visit http://jetpack.cloud.localhost:3000/pricing?flags=-jetpack/pricing-page-rework-v1,jetpack/pricing-page-product-lightbox
- See/inspect the "**More about <product_name>**" links, immediately after each paid product description. (See screenshots below).
- Verify they look fine visually.
- Verify that the "**More about <product_name>**" links are showing only on purchasable products.
- Verify that clicking the "**More about <product_name>**" links open the lightbox for each product.
- Disable the lightbox feature flag by removing `,jetpack/pricing-page-product-lightbox` from the url - Or just visit this link: http://jetpack.cloud.localhost:3000/pricing?flags=-jetpack/pricing-page-rework-v1
- Verify that the "**More about <product_name>**" links are no longer showing on any of the product cards (when the lightbox feature-flag is not enabled).


#### Pre-merge Checklist



- [X] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)? **N/A**
- [X] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)? **N/A**
